index.html

```diff
@@ -51,6 +51,147 @@ <h2>I am a Software Developer</h2>
+            <p class="he">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ul">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="button2">see project</button>
+          </div>
+          <div class="name2">
+            <h2 class="hd">Professional Art Printing Data</h2>
+            <p class="he">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ul">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="button2">see project</button>
+          </div>
+
+          <div class="name2">
+            <h2 class="hd">Professional Art Printing Data</h2>
+            <p class="he">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ul">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="button2">see project</button>
+          </div>
+          <div class="name2">
+            <h2 class="hd">Professional Art Printing Data</h2>
+            <p class="he">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ul">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="button2">see project</button>
+          </div>
+          <div class="name2">
+            <h2 class="hd">Professional Art Printing Data</h2>
+            <p class="he">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ul">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="button2">see project</button>
+          </div>
+        </div>
+      </section>
+      <section>
+        <div class="header3">
+          <h2 class="hf">About Me</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="button3">Get My Resume</button>
+         <div class="bckimg"> <img src="img2/Group (3).png" alt=""></div>
```

Comment:
- [ ] I kindly suggest that you use the `background-image` property through your `style.css` to apply your background images.

Kindly check the video below to know how to download your background image:


https://user-images.githubusercontent.com/52934128/219375621-71d4d0ca-fa5e-4bd8-9732-22c25ffc63ae.mp4